CHANGES.rst

```diff
@@ -26,6 +26,11 @@ align_refs
   MIRI coronagraphy data, due to large contiguous regions of NON_SCIENCE
   pixels [#6722]
 
+ami
+---
+
+- Added functionality for AmiAverageStep use in CLI environment [#6797]
```

Comment:
Style suggestion: The title of the PR is actually a bit more descriptive of the actual change.